google/provider_test.go

```diff
@@ -203,7 +203,7 @@ func TestAccProviderUserProjectOverride(t *testing.T) {
 				Check: func(s *terraform.State) error {
 					// The token creator IAM API call returns success long before the policy is
 					// actually usable. Wait a solid 2 minutes to ensure we can use it.
```

Comment:
Please update the comment.
`Wait a solid 2 minutes` --> `Wait a solid 5 minutes`

Reply:
This PR was a test and will not be merged.